### PR TITLE
feat: en francais [no issue]

### PR DESCRIPTION
**Pull request by christophehurpeau**
### Contexte

On écrit nos descriptions de PR en francais, pourquoi avoir un template en anglais ?

#### Options:
- [ ] This PR is a feature branch
- [ ] Auto merge with `[skip ci]`
- [ ] Auto merge when this PR is ready and has no failed statuses. (Also has a queue per repo to prevent multiple useless "Update branch" triggers)
- [x] Automatic branch delete after this PR is merged

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-<h3>Testing plan</h3>
-- [ ] Test this
-- [ ] Test that
+<!-- Décommente si tu as besoin d'un plan de test
+<h3>Plan de test</h3>
+- [ ] Tester ceci
+- [ ] Tester cela
 -->
 
 <!-- do not edit after this -->
```
Comment (Mxime) [inline pull_request_template.md]: Ces commentaires aussi, alors ?

File: pull_request_template.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-If needed, explain here the solution you chose for this
+Si besoin, explique ici la solution que tu as choisis
 
-<!-- Uncomment this if you need a testing plan
-<h3>Testing plan</h3>
-- [ ] Test this
-- [ ] Test that
+<!-- Décommente si tu as besoin d'un plan de test
+<h3>Plan de test</h3>
```
Comment (Mxime) [inline pull_request_template.md]: Profitons-en pour utiliser `###` ?

File: pull_request_template.md (hunk 1 above)
Comment (christophehurpeau) [reply pull_request_template.md]: ceux là on peut pas

Comment (christophehurpeau) [reply pull_request_template.md]: https://github.com/christophehurpeau/reviewflow/blob/master/src/pr-handlers/actions/utils/parseBody.ts#L3-L4

**Pull request closed without merging**